### ISSUE TITLE
Не добавляется текстовая метка на график

### ISSUE THREAD

**Issue by atrishin84** [SBProX/SBProX-Dev#1503]
При добавлении текстовой метки на график ничего не происходит. Порядок действий согласно инструкции:
https://www.manula.com/manuals/sbprox/sbprox-manual/1/ru/topic/3-5-3-4
"Чтобы добавить на график текст, выберите значок с изображение буквы “A” на панели Инструменты или воспользуйтесь горячей клавишей T. Щелкните левой кнопкой мыши в область на графике, куда необходимо добавить текст. В окне Settings можно поменять Цвет текста, изменить саму надпись в поле “Текст”, изменить Размер добавленного текста, Удалить (либо клавиша Del на клавиатуре), “Ок” – применить выбранные настройки."
Ожидаемый результат: на графике должна появиться текстовая метка
Фактический результат: текстовая метка на графике не появляется

**Comment by ElizarovSS** [comment]
Все работает
[Пример.zip](https://github.com/SBProX/SBProX-Dev/files/5072650/default.zip)

**Comment by atrishin84** [comment]
согласен, что на вашем рабочем месте все работает, но на моем нет. ваша техническая поддержка подтвердила, что это баг и рекомендовала завести тикет здесь.

**Comment by atrishin84** [comment]
обнаружил самостоятельно, что проблема заключалась в настройке прозрачности цвета текстовой метки. получается если впервые используешь текстовую метку и в диалоговом окне не меняешь параметры цветы оставляя по умолчанию то прозрачность равна нулю и поэтому фон графика перекрывает метку. если изменить значение прозрачности на 100% текстовая метка отображается корректно.

**Comment by Janna-Janna** [comment]
Спасибо
Сегодня так же обнаружила. 




Отправлено из мобильной Почты Mail.ru


понедельник, 21 сентября 2020 г., 19:26 +0600 от notifications@github.com  <notifications@github.com>:

**Comment by ElizarovSS** [comment]
На одном ПК работает, на другом нет

**Comment by Janna-Janna** [comment]
??

  
>Четверг, 12 ноября 2020, 5:10 +03:00 от ElizarovSS <notifications@github.com>:
> 
> 
>На одном ПК работает, на другом нет
>—
>You are receiving this because you commented.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
С уважением Жанна Мин.

**Comment by ElizarovSS** [comment]
На Server2012 работает, на семерке профи нет

**Comment by Janna-Janna** [comment]
добрый день.
 
Не понятно. Какой сервер? семёрка? 
>Четверг, 12 ноября 2020, 13:34 +03:00 от ElizarovSS <notifications@github.com>:
> 
> 
>На Server2012 работает, на семерке профи нет
>—
>You are receiving this because you commented.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
С уважением Жанна Мин.

**Comment by ElizarovSS** [comment]
На Windows Server 2012 R2 работает, на Windows 7 Professional не работает.

**Comment by atrishin84** [comment]
![image](https://user-images.githubusercontent.com/69624266/99192122-2da5bb80-2782-11eb-832d-a57f9d194368.png)
